java/squeek/wailaharvestability/helpers/StringHelper.java

```diff
@@ -15,7 +15,7 @@
 	{
 		try
 		{
-			HarvestLevels = Class.forName("tconstruct.library.util.HarvestLevels");
+			HarvestLevels = Class.forName("slimeknights.tconstruct.library.utils");
```

Comment:
This is missing the class name, isn't it? Shouldn't this be `"slimeknights.tconstruct.library.utils.HarvestLevels"`?